### ISSUE TITLE
RtGistCommentTest.java:57-59: Apply this rule to all...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1377]
The puzzle `1306-59643c80` from #1306 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/cf3c5acb8a87c549250735e64398ff219237b198/src/test/java/com/jcabi/github/RtGistCommentTest.java#L57-L59

The puzzle was created by rultor on 05-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C9R53K5JA) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#1377](https://github.com/jcabi/jcabi-github/issues/1377) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1377](https://github.com/jcabi/jcabi-github/issues/1377) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1021328395); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by llorllale** [comment]
@0crat waiting on review for PR #1387

**Comment by 0crat** [comment]
> @0crat waiting on review for PR #1387 ([here](https://github.com/jcabi/jcabi-github/issues/1377#issuecomment-381336794))

@llorllale The impediment for [#1377](https://github.com/jcabi/jcabi-github/issues/1377) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `1306-59643c80` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1388](https://github.com/jcabi/jcabi-github/issues/1388) is still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1377](https://github.com/jcabi/jcabi-github/issues/1377) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1394](https://github.com/jcabi/jcabi-github/issues/1394) is still not solved; solved: [#1388](https://github.com/jcabi/jcabi-github/issues/1388).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1399](https://github.com/jcabi/jcabi-github/issues/1399) is still not solved; solved: [#1388](https://github.com/jcabi/jcabi-github/issues/1388), [#1394](https://github.com/jcabi/jcabi-github/issues/1394).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1404](https://github.com/jcabi/jcabi-github/issues/1404) is still not solved; solved: [#1388](https://github.com/jcabi/jcabi-github/issues/1388), [#1394](https://github.com/jcabi/jcabi-github/issues/1394), [#1399](https://github.com/jcabi/jcabi-github/issues/1399).

**Comment by 0pdd** [comment]
@0pdd all 4 puzzles are solved here: [#1388](https://github.com/jcabi/jcabi-github/issues/1388), [#1394](https://github.com/jcabi/jcabi-github/issues/1394), [#1399](https://github.com/jcabi/jcabi-github/issues/1399), [#1404](https://github.com/jcabi/jcabi-github/issues/1404).
